src/mscorlib/ref/mscorlib.cs

```diff
@@ -9991,41 +10000,6 @@ public sealed partial class InternalsVisibleToAttribute : System.Attribute
         public bool AllInternalsVisible { get { throw null; } set { } }
         public string AssemblyName { get { throw null; } }
     }
-    public static partial class IsBoxed
```

Comment:
Your change to move these to the VisualC contract should go to corefx first; or together with CoreCLR that has this change.